README.md

```diff
@@ -1,128 +1,207 @@
 ## [简体中文](/README_CN.md)
-# thprac
-A tool for practicing Touhou Shmups
-## [Download](https://github.com/touhouworldcup/thprac/releases/latest) ([latest beta](https://nightly.link/touhouworldcup/thprac/workflows/main/master/thprac.zip))
-[![ko-fi](https://ko-fi.com/img/githubbutton_sm.svg)](https://ko-fi.com/V7V7O03J4)
 
-# How to use
-Either: place the executable in the same folder as your game, and run it. The game must be updated to the latest version and the executable must have it's original filename (thXX.exe for most games, 東方紅魔郷.exe for Embodment of Scarlet Devil, alcostg.exe for Uwabami Breakers). Thprac will automatically detect vpatch. Steam versions will not work with this method.
+# **thprac**
+> thprac is a tool for practicing Touhou shmups.
 
-Or: Run the game in whatever way you want (be it through thcrap, with vpatch, through Steam, etc...), then run the thprac exe. Thprac will automatically detect the running game and attach itself to it. You may need to return to the main menu for thprac to take effect.
+[![ko-fi](https://ko-fi.com/img/githubbutton_sm.svg)](https://ko-fi.com/V7V7O03J4)
 
-Or: Use the launcher. This video explains how the launcher works.
+## **Table of Contents**
+* [Downloading](#downloading)
+* [Usage](#usage)
+* [Compatability](#compatability)
+* [Features](#features)
+* [Backspace Menu](#backspace-menu)
+* [Advanced Options](#advanced-options)
+* [Statistics](#statistics)
+* [Game Specific Features](#game-specific-features)
+* [Credits](#credits)
+* [FAQ](#faq)
+
+## <a name="downloading"></a> **Downloading**
+If you want to download the tool, you can download the [latest release](https://github.com/touhouworldcup/thprac/releases/latest) here and the [latest beta](https://nightly.link/touhouworldcup/thprac/workflows/main/master/thprac.zip) here.
+
+## <a name="usage"></a> **Usage**
+You can use this tool in a variety of ways; the main ways to do this are **placing the executable in the same folder as your game**, **running the executable after running your game**, or **using the thprac launcher**.
+
+### **Using the folder**
+**This method will NOT work with Steam games.** You can run the thprac tool by placing the `thprac.exe` file in the same folder as your game. This method will also automatically detect vpatch. You must ensure that the game is updated to the latest version and follows one of the following naming schemes. This method also works with vpatch.
```

Comment:
please refer to thprac like it's just a noun, and don't call it "the thprac tool"